md/URL-customization.md

```diff
@@ -0,0 +1,29 @@
+# URL customization
+
+You can request to customize your domain to have an URL that uses a domain different from the default one (* bonitacloud.com).
+
+:::info 
+**Note:** You will need to own the targeted domain and delegate its management to Bonita.
+:::
+
+## Requesting a URL customization
+
+Create a new support case to request a URL Customization of your Bonita Cloud. Make sure to provide the following information in your case:
+* The domain or sub-domain to delegate to Bonita
+
+:::info 
+**Note:** The same domain must be applied to all environments.
+:::
+
+* The current and new environment URL:
```

Comment:
If we speak about URL we should put in front the protocol (https), or we may use FQDN instead.